utils/calendrical_calculations/src/rata_die.rs

```diff
@@ -12,17 +12,21 @@ use core::ops::{Add, AddAssign, Sub, SubAssign};
 #[allow(unused_imports)]
 use core_maths::*;
 
-/// The *Rata Die*, or *R.D.*, or `fixed_date`: number of days since January 1, 1 CE.
+/// The *Rata Die*, or *R.D.*: number of days since January 1, 1 CE.
 ///
 /// See: <https://en.wikipedia.org/wiki/Rata_Die>
 ///
-/// It is a logic error to construct a RataDie
-/// except from a date that is in range of one of the official calendars.
+/// Typically, one should obtain RataDies from other calendrical code, rather than constructing them from integers.
+/// The valid range for direct construction is deliberately not documented as it may change.
```

Comment:
Nit: Since this is a re-exported type, it should get the re-export boilerplate similar to https://github.com/unicode-org/icu4x/pull/6372

Reply:
It's marked as no_inline in the reexport. We don't actually care about people being pointed to a canonical location in icu_calendar for it, unlike with the input types where we don't want them importing from the dependency crate.

We should check this does the right thing in the metacrate.

Reply:
I don't understand the line we're drawing, then? We just recently started re-exporting across crates, and it was part of our agreement that we make sure docs appropriately indicate the "real" home of a type when it is re-exported.